src/CommandLineUtils/HelpText/DefaultHelpTextGenerator.cs

```diff
@@ -23,6 +23,11 @@ public class DefaultHelpTextGenerator : IHelpTextGenerator
         /// </summary>
         protected DefaultHelpTextGenerator() { }
 
+        /// <summary>
+        /// Determines if commands are ordered by name in generated help text
+        /// </summary>
+        public bool OrderCommandsByNameInHelpText { get; set; } = true;
```

Comment:
One last request: can we name this `SortCommandsByName`? Once that's in, I'll be happy to merge this PR.

Reply:
Sure thing, coming up...